public/app.js

```
@@ -211,7 +211,64 @@ async function showSection(sectionName) {
     // Charger les données spécifiques à la section
     switch (sectionName) {
         case 'dashboard':
-            loadDashboard();
+            // Rechargement complet comme la première fois
+            console.log('🔄 Dashboard: Rechargement complet comme première visite');
+            
+            // 1. FORCER la réinitialisation COMPLÈTE de TOUTES les dates
+            const currentDate = new Date();
+            const currentYear = currentDate.getFullYear();
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)